<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>okno</class>
 <widget class="QMainWindow" name="okno">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>682</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <pointsize>18</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QStackedWidget" name="stackedWidget">
    <property name="geometry">
     <rect>
      <x>9</x>
      <y>9</y>
      <width>782</width>
      <height>511</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="Pixel">
     <widget class="QLineEdit" name="lineEdit_PixelCoordinatesX">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>110</y>
        <width>211</width>
        <height>41</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_PixelCoordinatesY">
      <property name="geometry">
       <rect>
        <x>360</x>
        <y>110</y>
        <width>211</width>
        <height>41</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_PixelCoordinatesX">
      <property name="geometry">
       <rect>
        <x>200</x>
        <y>60</y>
        <width>21</width>
        <height>51</height>
       </rect>
      </property>
      <property name="text">
       <string>X</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_PixelCoordinatesY">
      <property name="geometry">
       <rect>
        <x>460</x>
        <y>60</y>
        <width>21</width>
        <height>51</height>
       </rect>
      </property>
      <property name="text">
       <string>Y</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_PixelCoordinates">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>0</y>
        <width>191</width>
        <height>71</height>
       </rect>
      </property>
      <property name="text">
       <string>Координаты:</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_PixelColor">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>180</y>
        <width>101</width>
        <height>61</height>
       </rect>
      </property>
      <property name="text">
       <string>Цвет:</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_PixelColor">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>230</y>
        <width>251</width>
        <height>51</height>
       </rect>
      </property>
      <property name="text">
       <string>Выбрать цвет</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_PixelBrushWidth">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>340</y>
        <width>221</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>Ширина кисти:</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_PixelBrushWidth">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>400</y>
        <width>261</width>
        <height>41</height>
       </rect>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="Line">
     <widget class="QLineEdit" name="lineEdit">
      <property name="geometry">
       <rect>
        <x>190</x>
        <y>110</y>
        <width>491</width>
        <height>221</height>
       </rect>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="Rectangle">
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>140</y>
        <width>111</width>
        <height>231</height>
       </rect>
      </property>
      <property name="text">
       <string>vssv</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="checkBox">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>180</y>
        <width>70</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>CheckBox</string>
      </property>
     </widget>
     <widget class="QDialogButtonBox" name="buttonBox">
      <property name="geometry">
       <rect>
        <x>270</x>
        <y>210</y>
        <width>156</width>
        <height>23</height>
       </rect>
      </property>
      <property name="standardButtons">
       <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
      </property>
     </widget>
     <widget class="QCommandLinkButton" name="commandLinkButton">
      <property name="geometry">
       <rect>
        <x>480</x>
        <y>70</y>
        <width>172</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>CommandLinkButton</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="Circle">
     <widget class="QCalendarWidget" name="calendarWidget">
      <property name="geometry">
       <rect>
        <x>220</x>
        <y>140</y>
        <width>312</width>
        <height>183</height>
       </rect>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="Ellipse"/>
    <widget class="QWidget" name="Arc"/>
    <widget class="QWidget" name="Polyline"/>
   </widget>
   <widget class="QPushButton" name="pushButton_OK">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>540</y>
      <width>401</width>
      <height>45</height>
     </rect>
    </property>
    <property name="text">
     <string>OK</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
